src/docpage.js

```diff
@@ -3,19 +3,201 @@ import React from 'react'
 export default class DocPage extends React.Component {
 
   componentWillMount() {
-    let docUrl = window.location.href.split('?')[0]
-    let dataUrl = `${docUrl}/json`
-    $.get(dataUrl, (doc) => {
-      this.setState({doc})
+    let docUrl = this.props.docUrl || window.location.href.split('?')[0]
```

Comment:
Is there ever a case when `props.docUrl` is not set? If not, let's remove the `||` fallback.